walsamlee/politico
# #163690223 Users Login

#### What does this PR do?
Add `login` functionality to consume the `login endpoint` on the `UI` template
#### Description of Task to be completed?
Have `sign-in` page working on the `UI`template
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-users-login-163690223`
- Visit `https://walsamlee.github.io/politico/users/login.html` on the browser
#### What are the relevant pivotal tracker stories?
#163690223
